src/backend/distributed/transaction/citus_dist_stat_activity.c

```diff
@@ -155,7 +155,12 @@ FROM \
 	get_all_active_transactions() AS dist_txs(database_id, process_id, initiator_node_identifier, worker_query, transaction_number, transaction_stamp, global_pid) \
 	ON pg_stat_activity.pid = dist_txs.process_id \
 WHERE \
-	dist_txs.worker_query = false;"
```

Comment:
CitusInitiatedBackend becomes pretty much useless at this point. I tried to drop that structure. However, CitusInitiatedBackend keeps track of groupId whereas globalPID keeps track of NodeIds. So, the changes cascaded into citus_lock_waits, and I decided to handle that with a separate PR